src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,212 @@
+- Use the correct element to ensure the content communicates the intended meaning.
+
+Here are some benefits of using semantic HTML:
+
+- It allows UX writers and developers to add meaning to content so search engines, screen readers and web browsers can understand it.
+- It improves user experience. It's easier for engineers to understand and to develop with.
+- It helps create faster, responsive, and better mobile user experiences. It's easier to make responsive, have reduced file size, and load faster.
+- It helps web pages get better rankings. Search engines give more importance to those that have them.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Table of Contents
+
+<ul>
+  <li>
+    <a href="#html-elements">HTML Elements</a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#creating-elements-using-tags">Creating Elements Using Tags</a>
+    </li>
+    <li>
+      <a href="#common-html-elements">Common HTML Elements</a>
+    </li>
+  </ul>
+  <li>
+    <a href="#semantic-elements">Semantic Elements</a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#the-strong-element">The Strong Element</a>
+    </li>
+    <li>
+      <a href="#strong-vs-bold-vs-css-styling">Strong vs Bold vs CSS Styling</a>
+    </li>
+    <li>
+      <a href="#the-emphasis-element">The Emphasis Element</a>
+    </li>
+    <li>
+      <a href="#emphasis-vs-italic">Emphasis vs Italic</a>
+    </li>
+  </ul>
+  <li>
+    <a href="#accessibility">Accessibility</a>
+  </li>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## HTML Elements
+
+HTML elements wrap content and describe meaning, structure, or context to web browsers, developers, and assistive technologies.
+
+> For more information, visit <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element"> Mozilla's Developer Documentation - HTML elements</a>.
+
+As a UX Writer, it's important to:
+
+- **Understand** what is the purpose of the enclosed text.
+- **Know** what element to use depending on the purpose of the text.
+- **Ensure** we are communicating the intended meaning of the enclosed text to the users.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Creating Elements Using Tags
+
+Use **HTML tags** to create an element. These tags represent the start and end of a piece of content.
+
+An element consists of three main parts:
+
+- An **opening tag** that represents the start of the paragraph element.
+- **Content** within the tags.
+- A **closing tag** that represents where the element ends.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Example</h3>
+
+`<p>Download the Rocket Homes mobile application on your mobile device to start looking for your dream home.</p>`
```

Comment:
![image](https://user-images.githubusercontent.com/4342363/123142626-a4491c80-d427-11eb-89fd-d824ca2decd6.png)

Using one backtick is for single line codes. If it's too long, it pushes all the content and causes weird scrolling issues on small viewports.

Write it like this 
![image](https://user-images.githubusercontent.com/4342363/123142813-d8bcd880-d427-11eb-800f-616a0ddde292.png)

and test it on a small viewport.